models/discount-code/src/discount-code-draft/generator.ts

```diff
@@ -12,7 +12,7 @@ const generator = Generator<TDiscountCodeDraft>({
   fields: {
     name: fake(() => LocalizedStringDraft.random()),
     description: fake(() => LocalizedStringDraft.random()),
-    code: fake((f) => f.lorem.word()),
```

Comment:
WDYT of instead of using `f.lorem.words({ min: 1, max: 3 })` we use `f.string.alphanumeric({ length: 20 })` to reduce collision probability even more?.

Reply:
or increasing the minimum to at least 2?

Reply:
I do like the alphanumeric idea. I'll make that change now. Thanks!